SCRIPTS/resource.VCFparser.pl

```diff
@@ -902,7 +991,7 @@
 print STDERR "\n";
 print STDERR "++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++\n";
 print STDERR "+ The MIT License (MIT)                                                                  +\n";
-print STDERR "+ Copyright (c) 2016-2022 Sander W. van der Laan & Vinicius Tragante dó Ó                +\n";
+print STDERR "+ Copyright (c) 2016-2017 Sander W. van der Laan & Vinicius Tragante dó Ó                +\n";
```

Comment:
please update 2017 > 2022

Reply:
I've already tested 1Gp3.
Do I need to change 1Gp1 and GoNL5 or can they leave them as they are?

Reply:
I get these  `*** WARNING *** Could not find the allele frequency for [ rs112591207 ]. Check your reference-file.` but that's because of the reference file. For the rest it at all works.

Reply:
Great. Which reference file is this?

Reply:
ALL.wgs.phase3_shapeit2_mvncall_integrated_v5b.20130502.sites.vcf.gz (subset)

I don't know what the reference file for 1Gp3+GoNL5 should be.

Reply:
This option works given the proper input - it's not in the `RESOURCES` folder.